### ISSUE TITLE
Radiant Dash Feedback / Issues Thread

### ISSUE THREAD

**Issue by ProbablePrime** [Frooxius/NeosPublic#727]
Congrats on shipping the dash. Here's the minor items I mentioned. These are all Minor.

- [ ] Status Facet Colouring leaves the currently selected status as too ambiguous. Its hard for me as a colorblind user to see what item is selected.
- [ ] Save world location is not idea. Saving worlds is somewhat complicated as it is and this makes it more complicated :( As a new user this may be confusing but doesn't block me as an experienced user.
- [ ] Exit button options are still confusing. Congrats on redesigning it to be bolder and clearer but I still don't feel that users know what the difference between the two buttons are.
- [ ] Exit button on dash may make users think it closes the dash.
- [ ] Create new world button is too small and invisible for new users
- [ ] Dash options require a full trigger pull. Some users may find this undesirable
- [ ] Contrast and color issues, are basically everywhere. I'm colorblind, we have legally blind users and all sorts of other issues. I'd be happy to sit or record a full video on accessibility problems here.
- [ ] As a new user, i'm going to struggle to close the dash. Let's see what the feedback here brings with your changes late last night thanks :)
- [ ] I heard there were going to be UI Sounds added to the dash in the future. As a suggestion could you add another volume bar to the sound settings for UI. So we'd have Sound Effects, Multimedia and UI.
- [ ] Popping out various UIs, including the inventory and session menu was very useful when in the role of a session host or managing users entering a meeting/party.

Once again these are all minor and some are being addressed as the dash rolls out. I want to log them so that they are not forgotten.

**Comment by sirkitree** [comment]
I agree with many of these. Here's my top issue (if I'm permitted to pile on):
- [ ] Suggest that Save become it's own tab like it was on the old toolbelt menu, or combine world operations (Save, Save as, Leave, Discard, Close, etc) on the Worlds tab.
- [ ] Save world screen should have the 'Sync' indicator on there. Many times I want to see if my world is synchronized after saving which indicates that it's *actually* saved. Currently I must switch to the home tab to see this.

**Comment by Frooxius** [comment]
Thanks for feedback! One thing I should note is that things are in flux right now. As more pieces of the UI are reworked, they'll be moved around into better places, but it's going to be a bit weird in some places while it's a mix of old and new. Also as the pieces are reworked, you'll be able to rearrange the UI to suit you and your workflow better.

For the Status facet, I'll have Coffee make a better version of it with icons and so on, so that should improve soon!

I'm not sure where exactly would be the be options for saving world. On the home screen they don't make as much sense, since it's more of a thing for building, so I don't want to put them there. Having them in a place that relates to the open world made most sense for now.

For the exit screen, I do plan on having it actually list worlds the user has open and asking if any of those should be saved. Essentially the equivalent of "These are your unsaved documents" screen. But for now it's just these two options.

I'm not fully sure what to do about users thinking the exit button closes the dash. Any suggestions there? Is this going to be a big problem? If user clicks it, they'll see "Exit Neos" screen, so that should clear any confusion quickly anyways.

For the trigger pull, that's not too specific to the dash, but UIX in general. Dash is using those systems, that might be better as a separate general issue to provide partial-pull interaction for UIX.

UI sound category sounds good! That'll be easy to add too.

And the popping out is going to be naturally possible with facets! The old inventory is still there if you want to share that one. Session menu wasn't poppable before though. I can add buttons to pop the old world browser though if that's commonly used for now.

**Comment by Frooxius** [comment]
Oh I also completely forgot, the contract and color issues video would be very helpful! I'm not fully sure which parts are going to be problematic and which won't be.

One thing I'd like to add later on is support for the WCAG 2.0 guidelines that you showed me a while back. That way it's supported at the code level when UIX styling is implemented, so it can be enforced automatically by having it simply adjust the provided colors.

**Comment by Frooxius** [comment]
I think I have a good idea for the open/close dash problem. What I'd do is make a simple button facet, that's going to be pinned to a permanently visible location in the user's playspace (aligned with the user, so it's always in front of them). It will allow opening and closing the dash.

It will always be present, even if the dash is closed. Once the user learns the button and wants it to go away, they simply enable the UI edit mode and remove it, so advanced users won't be bothered by it all the time. What do you think?

**Comment by shiftyscales** [comment]
That could work- I think having a button like that will eventually prove mandatory anyway if we want to have touch/hand-only interfaces, e.g. using the Leap, or Oculus hand-tracking as an input method.

I do think there will likely be some new users that will want it hidden though (especially if it's always visible), but aren't necessarily going to want to adjust much else about their UIs. Especially if it's the only thing that's visible when the dash is closed as it'd lower immersion.

**Comment by Reiko9** [comment]
**PLEASE....** Just for a Comfortable Impression to New-users make the Dash window Slide Up and Down to Minimize/maximize whenever users Toggle it with a Button press.
    In other words say if the Inventory  button is Selected and "Glowing-Blue" and I click the "Glowing-Blue" Inventory button, then the GUI window slides down and is hidden.. (Except the Toolbar with its buttons)

While the Context menu press on the Left-side of my head to Hide the Entire GUI works, if I am working on a world and Wish to see message notifications on my friends list and quickly access my inventory I need the Toolbar visible.

New-users will not know about the Context menu held to Side-of-head shortcut and will wonder how to Minimize the GUI that is covering their view...

Also it would be Great if we could Enable seeing Messages on Tiles from Friends in a Similar way to how we can see whenever a friend Logs-in. (That Green tile with White text that floats in front of you)   (As of right now I only know if someone sent me a message from the friends list button...)

**Comment by Reiko9** [comment]
Clapping Hands (Gently preferably) can Open & Close a GUI on systems with No Buttons...  Other more complex hand-gestures could work, but this one is easy to explain across many cultures and languages...
https://www.youtube.com/watch?v=9y1yr9NgoP8&feature=youtu.be
Leapmotion Demonstrated back in 2016 that even a ThumbsUp Gesture could be reliably detected and used for something like opening a GUI....

**Comment by shiftyscales** [comment]
You don't need to use the gesture in which you hold the controller up to your head any longer, @Reiko9. That was the purpose of introducing the dash in its current form. For Oculus Touch/Index users, you have a dedicated button to open/close the dash. For Vive/WMR, it is a long-press of the context menu to do so.

Once workspaces are added, and users can create facet containers, and place them on their avatar, etc. it'll be possible to have always-active bits of UI that could expand when interacted with if that's the desire of how a user wants to configure/customize their UI.

Designing the UI to be interacted with entirely controller-free would be something that could come at a later date, and isn't related directly to the dash itself.

**Comment by Reiko9** [comment]
@shiftyscales I was trying to convey... That on a system with No Physical Buttons... Like (Leapmotion/Quest hand tracking) you could use a Hand-Gesture to Open/Close the GUI so you do not need a floating button on your Vision to open it...  Some users may prefer a button on their view and some will want their vision entirely clean of gui elements at times..
    Thankfully users do not demand this functionality yet... sorry for the confusion...

**Comment by shiftyscales** [comment]
I understood that point, @Reiko9. I'm just saying that is a separate issue/isn't directly related to this thread/GitHub issue. That'd be something you should request separately (more specifically just controller-free/Leap-only control options for Neos rather than specifying any one piece.)
